docs/ipxe_home_server.md

```diff
@@ -0,0 +1,57 @@
+# iPXE on home server #
+
+This guide is about setting up an iPXE-tftp server on home server machine.
+
+## Installation ##
+
+1. Server should be running inside a Docker container therefore docker is
+required. To install docker on Debian go [here](https://docs.docker.com/engine/installation/linux/debian/).
```

Comment:
@leszekbulawa it would be enough to point to installation page. Distro should be choosen by user.